src/dune/file_tree.ml

```diff
@@ -35,14 +35,13 @@ module Dune_file = struct
   module Plain = struct
     type t =
       { path : Path.Source.t
-      ; sub_dirs : Predicate_lang.Glob.t Sub_dirs.Status.Map.t
-      ; mutable sexps : Dune_lang.Ast.t list
+      ; contents : Sub_dirs.Dir_map.per_dir
+      ; for_subdirs : Sub_dirs.Dir_map.t
       }
 
     let get_sexp_and_destroy t =
-      let sexps = t.sexps in
-      t.sexps <- [];
-      sexps
+      (* TODO add back GC *)
```

Comment:
Do you wan to add it back in this PR, or do you intend to keep it for indefinite future?
If the latter, can you write a comment explaining what the challenge is?